packages/dds/matrix/src/matrix.ts

```diff
@@ -127,18 +133,20 @@ export class SharedMatrix<T extends Serializable = Serializable>
     /**
      * {@inheritDoc @fluidframework/datastore-definitions#IChannelFactory.create}
      */
-    public static create<T extends Serializable = Serializable>(runtime: IFluidDataStoreRuntime, id?: string) {
+    public static create<T>(runtime: IFluidDataStoreRuntime, id?: string) {
         return runtime.createChannel(id, SharedMatrixFactory.Type) as SharedMatrix<T>;
```

Comment:
```suggestion
        return runtime.createChannel(id, SharedMatrixFactory.Type) as SharedMatrix<Serializable<T>>;
```

Reply:
Interesting.  If we make the DDS ctor private and require everyone to use SharedMatrix.create<T>, we could apply the constraint once here instead of sprinkling Serializable<T> into every member signature.

What do you think?

Another variant I considered was:
```ts
class SharedMatrix<T, TItem=Serializable<T>> { ... }
```
...and then using TItem everywhere instead of T.  The downside was that someone could provide an incorrect TItem.

Reply:
I guess there is a downside, which is that we lose 'Serializable<T>' in API signatures in the documentation, which is probably an important clue when deciphering `'value' in setCell(..., value) cannot be assigned to 'never'' errors.

Darn.   That would have saved me a lot of typing.